public/config/new_config.json

```diff
@@ -0,0 +1,114 @@
+{
+    "widgets": [
+        {
+            "type": "socketState",
+            "id": 1,
+            "label": "wallPow",
+            "position": {
+                "my": "left top",
+                "at": "left top"
+            },
+            "format": {
+                "trueText": "Charger has power",
+                "trueColor": "#75ff75",
+                "falseText": "No charger power",
+                "falseColor": "#ff6666"
+            },
+            "data": {
+                "topic": "telemetry",
+                "topicType": "rq_msgs/msg/Telemetry",
+                "topicDirection": "subscribe",
+                "topicAttribute": "charger_has_power"
+            }            
+        },{
+            "type": "socketValue",
+            "id": 3,
+            "label": "batteryVoltage",
+            "position": {
+
+            },
+            "format": {
+                "prefix": "Battery V ",
+                "suffix": "",
+                "textColor": "#000000",
+                "precision": 6
+            },
+            "data": {
+                "topic": "telemetry",
+                "topicType": "rq_msgs/msg/Telemetry",
+                "topicDirection": "subscribe",
+                "topicAttribute": "battery_v"
+            }
+        },{
+            "type": "socketButton",
+            "id": 11,
+            "label": "chargerOn",
+            "position":{
+                "my":"left top",
+                "at":"left top"
+            },
+            "format":{
+                "text": "Set charger ON"
+            },
+            "data":{
+                "service": "control_hat",
+                "serviceAttribute": "set_charger",
+                "serviceType": "rq_msgs/srv/Control",
+                "clickValue": "ON"
+            }
+        },{
+            "type": "socketJoystick",
+            "id": 14,
+            "label": "robotVelocity",
+            "position": {
+                "my":"left top",
+                "at":"left top"
+            },
+            "format": {},
+            "data": {
+                "topic": "cmd_vel",
+                "topicDirection": "publish",
+                "topicType": "geometry_msgs/msg/TwistStamped",
+                "topicAttribute": ["twist.angular.z", "twist.linear.x"],
+                "scale": [3.14, 2.0]
+            }
+        },
+        {
+            "type": "socketSlider",
+            "id": 16,
+            "label": "camera_pan",
+            "position": {
+                "my":"left top",
+                "at":"left top"
+            },
+            "format": {
+                "min": "0",
+                "max": "180",
+                "step": "1",
+                "default": "90",
+                "orientation": false
+            },
+            "data": {
+                "topic": "servos",
+                "topicDirection": "publish",
+                "topicAttribute": "name",
```

Comment:
This attribute name is incorrect. We can review how best to handle slider data for the ServoAngles topicType, then I'll make simplifications in the NodeJS code and you can implement the browser code.

Reply:
updated to an array